src/language-js/printer-estree.js

```diff
@@ -1610,15 +1610,8 @@ function printPathNoParens(path, options, print, args) {
     case "NumericLiteral": // Babel 6 Literal split
       return printNumber(n.extra.raw);
     case "BigIntLiteral":
-      return concat([
-        printNumber(
-          n.extra
-            ? n.extra.rawValue
-            : // TypeScript
-              n.value
-        ),
-        "n"
-      ]);
+      // babel: n.extra.raw, typescript: n.raw
+      return (n.extra ? n.extra.raw : n.raw).toLowerCase();
```

Comment:
Lowercasing is the only part of `printNumber` that applies to bigints. So I removed the `printNumber` call.